src/linagora.esn.videoconference.calendar/app/components/calendar-videoconference-form/calendar-videoconference.less

```diff
@@ -13,3 +13,40 @@
     flex-grow: 1;
   }
 }
+
+.conference-btns {
+  margin-left: 10px;
+  min-width: 0;
+}
+
+.btn-row {
+  display: flex;
+
+  a {
+    display: block;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  i {
+    padding-right: 15px;
+  }
+
+  .btn-link {
+    margin-left: 15px;
+  }
+}
+
+@media screen and (max-width: 990px) {
+  .conference-btns {
+    margin-left: 60px;
+  }
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
We have a LESS variable for this. Can you please take your time to find it?

Reply:
it was `@screen-xs-min` in `@import '~esn-frontend-common-libs/src/frontend/css/styles.less';`